src/utils/ButtonWithIcon.vue

```diff
@@ -131,6 +123,9 @@ export default {
         background: none;
         border: none;
     }
+    &-w5 {
+        width: 5em;
+    }
```

Comment:
Shouldn't that be in the DrawingToolboxButton component? As this style is scoped I think this will not be applied if the class is not applied in this component.

Reply:
This works. But thank you for the question! Because I thought having the styles in the DrawingToolboxButton wouldn't work because of the scoping. I very much prefer to have the styles with the toolbox instead of the icon-button. I'll change it 😎

Reply:
Done 😃